include/magma/vk/commandBuffer.h

```diff
@@ -10,9 +10,14 @@ class CommandBufferArr
 public:
     CommandBufferArr() = default;
     CommandBufferArr(VkDevice device, VkCommandPool commandPool, uint32_t count);
+    ~CommandBufferArr();
+
     void allocate(VkDevice device, VkCommandPool commandPool, uint32_t count);
     VkCommandBuffer beginCmdBuf(uint32_t i);
     void endCmdBuf(uint32_t i);
+    void _syncEndAndSubmitCmdBuf(uint32_t i, VkQueue queue);
```

Comment:
"_sync" was supposed to be a suffix. Not prefix.
It should be
```
endAndSubmitCmdBuf_sync
```
or
```
endAndSubmitCmdBufSync
```

Reply:
Done